app/cdap/components/FetchIconsFromWidget/Widget.tsx

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
Change the component (folder) name to WidgetSVG

Reply:
Done.